runtime: editorial improvements for c8d and wasm (#16266)
docker/docs: desktop/wasm/index.md

toc_max: 3
---

> Wasm (short for WebAssembly) is a faster, lighter alternative to the Linux &

Reviewer: I think we preferred the language "fast, light alternative" rather than faster/lighter

   COPY --from=build /build/hello_world.wasm /hello_world.wasm
   ENTRYPOINT [ "hello_world.wasm" ]
   ```

3. Build and push the image specifying the `wasi/wasm32` architecture. Buildx
   makes this easy to do in a single command.

   ```console
   $ docker buildx build --platform wasi/wasm32 -t username/hello-world .
   ...
   => exporting to image                                                                             0.0s
   => => exporting layers                                                                            0.0s
   => => exporting manifest sha256:2ca02b5be86607511da8dc688234a5a00ab4d58294ab9f6beaba48ab3ba8de56  0.0s
   => => exporting config sha256:a45b465c3b6760a1a9fd2eda9112bc7e3169c9722bf9e77cf8c20b37295f954b    0.0s
   => => naming to docker.io/username/hello-world:latest                                            0.0s
   => => unpacking to docker.io/username/hello-world:latest                                         0.0s
   $ docker push username/hello-world
   ```

## Troubleshooting

This section contains instructions on how to resolve common issues.

### Unknown runtime specified

If you try to run a Wasm container before you have opted in to use the

> containerd image store, you will see an error similar to the following:

Reviewer 2: ```suggestion
containerd image store,  an error similar to the following displays:
```
